t/unit/test_models.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import, unicode_literals
+
+import importlib
+
+from django.test import TestCase
+
+from django_celery_beat.models import PeriodicTask
+
+
+class ModelMigrationTests(TestCase):
+    def test_periodic_task_name_max_length_defaults_to_200_in_model(self):
```

Comment:
can we use `override_settings` decorator and add a test where `DJANGO_CELERY_BEAT_NAME_MAX_LENGTH = 191` as well? Rest LGTM.

Reply:
hey, thx for review. i tried that and it didn't work. it seemed like the setting had already been evaluated before the test could override it. so it always remained at 200. i also tried to put the model import inside of the test function and that didn't help either. it's possible of course to duplicate the whole test project and change the setting globally. do you think, this would be a good solution?

Reply:
i think i made it work with module reloading. reloading django models is not recommended to do, so i had to run the model tests in their own process.

Reply:
it works :-) https://travis-ci.com/tuky/django-celery-beat/jobs/177078955